--- old/data/profiles/meson.build
+++ new/data/profiles/meson.build
@@ -49,7 +49,7 @@
 
 generated_iccs = []
 foreach arg: icc_profiles
-  xml_i18n = i18n.merge_file(
+  i18n.merge_file(
     input: arg + '.iccprofile.xml',
     output: arg + '.iccprofile.xml',
     type: 'xml',
@@ -57,7 +57,7 @@
     po_dir: join_paths(meson.source_root(), 'po')
   )
   generated_icc = custom_target(arg + '.icc',
-    input: xml_i18n,
+    input: arg + '.iccprofile.xml',
     output: arg + '.icc',
     command: [ cd_create_profile, '--output=@OUTPUT@', '@INPUT@' ],
     install: true,
