--- a/configure.ac
+++ b/configure.ac
@@ -28,15 +28,6 @@
 
 AM_SILENT_RULES([yes])
 
-GETTEXT_PACKAGE=AC_PACKAGE_NAME
-AC_SUBST(GETTEXT_PACKAGE)
-AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE, ["$GETTEXT_PACKAGE"], [Define the gettext package to be used])
-
-AM_GNU_GETTEXT([external])
-# FIXME: Remove AM_GNU_GETTEXT_VERSION once autoreconf supports REQUIRE_VERSION
-AM_GNU_GETTEXT_VERSION([0.19.6])
-AM_GNU_GETTEXT_REQUIRE_VERSION([0.19.6])
-
 AC_SUBST(CFLAGS)
 AC_SUBST(CPPFLAGS)
 AC_SUBST(LDFLAGS)
--- a/data/Makefile.am
+++ b/data/Makefile.am
@@ -35,7 +35,7 @@
 	$(NULL)
 
 %.desktop: %.desktop.in
-	$(AM_V_GEN)$(MSGFMT) --desktop --template $< -d $(top_srcdir)/po -o $@
+	$(AM_V_GEN)cp $< $@
 
 %.service: %.service.in
 	$(AM_V_GEN)sed -e "s|@bindir[@]|$(bindir)|" $< > $@
