--- a/src/libical/icalrecur.c
+++ b/src/libical/icalrecur.c
@@ -143,6 +143,7 @@
 #include <stdlib.h>
 
 #if defined(HAVE_LIBICU)
+#include <stdbool.h>
 #include <unicode/ucal.h>
 #include <unicode/ustring.h>
 #define RSCALE_IS_SUPPORTED 1
@@ -1018,7 +1019,7 @@
 
     calendars = icalarray_new(sizeof(const char **), 20);
 
-    en = ucal_getKeywordValuesForLocale("calendar", NULL, FALSE, &status);
+    en = ucal_getKeywordValuesForLocale("calendar", NULL, false, &status);
     while ((cal = uenum_next(en, NULL, &status))) {
         cal = icalmemory_tmp_copy(cal);
         icalarray_append(calendars, &cal);
@@ -1411,7 +1412,7 @@
         }
 
         /* Check if specified calendar is supported */
-        en = ucal_getKeywordValuesForLocale("calendar", NULL, FALSE, &status);
+        en = ucal_getKeywordValuesForLocale("calendar", NULL, false, &status);
         while ((cal = uenum_next(en, NULL, &status))) {
             if (!strcmp(cal, rule.rscale)) {
                 is_hebrew = !strcmp(rule.rscale, "hebrew");
