--- a/data/CMakeLists.txt
+++ b/data/CMakeLists.txt
@@ -10,7 +10,7 @@
 	@ONLY
 )
 
-intltool_merge(${CMAKE_CURRENT_BINARY_DIR}/org.gnome.Evolution-alarm-notify.desktop.in org.gnome.Evolution-alarm-notify.desktop --desktop-style --utf8)
+configure_file(${CMAKE_CURRENT_BINARY_DIR}/org.gnome.Evolution-alarm-notify.desktop.in org.gnome.Evolution-alarm-notify.desktop COPYONLY)
 
 install(FILES ${CMAKE_CURRENT_BINARY_DIR}/org.gnome.Evolution-alarm-notify.desktop
 	DESTINATION ${autostartdir}
@@ -36,7 +36,7 @@
 
 set(BUILT_SCHEMAS)
 foreach(_schema IN LISTS SCHEMAS)
-	intltool_merge(${_schema}.in ${_schema} --no-translations --xml-style --utf8)
+	configure_file(${_schema}.in ${_schema} COPYONLY)
 	list(APPEND BUILT_SCHEMAS ${CMAKE_CURRENT_BINARY_DIR}/${_schema})
 endforeach(_schema)
 
--- a/src/services/evolution-source-registry/CMakeLists.txt
+++ b/src/services/evolution-source-registry/CMakeLists.txt
@@ -48,7 +48,7 @@
 
 set(builtin_sources)
 foreach(_file ${builtin_sources_files})
-	intltool_merge(${CMAKE_CURRENT_SOURCE_DIR}/builtin/${_file}.in ${CMAKE_CURRENT_BINARY_DIR}/builtin/${_file} -d -u)
+	configure_file(${CMAKE_CURRENT_SOURCE_DIR}/builtin/${_file}.in ${CMAKE_CURRENT_BINARY_DIR}/builtin/${_file} COPYONLY)
 	list(APPEND builtin_sources ${CMAKE_CURRENT_BINARY_DIR}/builtin/${_file})
 endforeach(_file)
 
